src/test/compile-fail/lint-exceeding-bitshifts.rs

```diff
@@ -11,7 +11,7 @@
 #![deny(exceeding_bitshifts)]
 #![allow(unused_variables)]
 #![allow(dead_code)]
-#![feature(num_bits_bytes)]
+#![feature(num_bits_bytes, const_indexing)]
```

Comment:
it compiles if the lint is set to allow: https://play.rust-lang.org/?gist=a4a9ec25dbc49a18f272&version=nightly